src/migrations/20200103080032-create-preferences.js

```diff
@@ -0,0 +1,33 @@
+
+const preferenceMigrations = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Preferences', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    userId: {
+      type: Sequelize.INTEGER
+    },
+    emailNotification: {
```

Comment:
Variables that contain a boolean as a value should be prefixed with either `is` or `has`